interop/client/client.go

```diff
@@ -34,7 +35,8 @@ import (
 
 var (
 	caFile                = flag.String("ca_file", "", "The file containning the CA root cert file")
-	useTLS                = flag.Bool("use_tls", false, "Connection uses TLS if true, else plain TCP")
+	useTLS                = flag.Bool("use_tls", false, "Connection uses TLS if true")
+	useALTS               = flag.Bool("use_alts", false, "Connection uses ALTS if true (this option can only be used on GCP)")
```

Comment:
Is there a spec for the interop binaries and their flags?  Or are the flags language-specific?

If the latter, it may be preferable to have a single "security" option that could be "none", "tls", or "alts", to avoid the illegal combinations of flags messiness.

If this needs to work this way, then LGTM.

Reply:
After talking offline this version LGTM -- it's not worth it due to needing to change the interop framework, which is in another repo.

Reply:
Thanks Doug. I just rebased this PR. It's ready now.